packages/ketcher-react/src/script/ui/component/view/Atom/Atom.jsx

```diff
@@ -14,11 +14,16 @@
  * limitations under the License.
  ***************************************************************************/
 
-function Atom({ el, shortcut, className, ...props }) {
+import classes from '../../../views/toolbars/ToolbarGroupItem/ActionButton/ActionButton.module.less'
```

Comment:
All styles related to Atom component should de defined inside Atom.module.less. Please do  not reuse styles of ActionButton component